spec/models/post_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  first_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+  subject { Post.create(author: first_user, title: 'Hello', text: 'This is my first post') }
+  it 'should be inheritanc of' do
+    expect(Post < ApplicationRecord).to eq(true)
+  end
+  before { subject.save }
+  it 'title should be equle to Hello' do
+    expect(subject.title).to eq('Hello')
+  end
+
+  it 'text should be given strirng' do
+    expect(subject.text).to eq('This is my first post')
+  end
+
+  it 'author shold be ' do
+    expect(subject.author).to eq(first_user)
+  end
```

Comment:
- [ ] Kindly correct your `it` blocks it hard to understand what your test cases do